core/src/PrognosticData.cpp

```diff
@@ -51,6 +51,13 @@ void PrognosticData::configure()
     tryConfigure(pDynamics);
 
     tryConfigure(iceGrowth);
+
+    // clang-format off
+    fieldsToCheck.push_back({"m_conc",   &m_conc,   std::make_pair(0, 1+1e4)});
+    fieldsToCheck.push_back({"m_damage", &m_damage, std::make_pair(0, 1+1e-4)});
+    fieldsToCheck.push_back({"m_snow",   &m_snow,   std::make_pair(0, 10)});
+    fieldsToCheck.push_back({"m_thick",  &m_thick,  std::make_pair(0, 50)});
```

Comment:
Use the names names in `gridNames.hpp`.
Use the `ModelArrayRef` system, since the model has infrastructure for mapping names to `ModelArray` fields.
Use an initializer list, rather than `std::make_pair`.